### PR TITLE
Person, student, teacher classes

### PR DESCRIPTION
**Pull request by chepkok3**
- [ ] Created class Person with:
- Instance vars: @id, @name, and @age.
- Constructor with name, age, and parent_permission as parameter. name and parent_permission 
- Getters for @id, @name, and @age.
- Setters for @name and @age.
- Private method of_age? that returns true if @age is greater or equal to 18 and false otherwise.
- Public method can_use_services? that returns true if person is of age or if they have permission from parents.
- [ ] Created class Student with: 
- Inherits from Person.
- Constructor extends parent's constructor by adding @classroom and a parameter for it.
- Method play_hooky that returns "¯\(ツ)/¯".
- [ ] Created class Teacher with: 
- Inherits from Person.
- Constructor extends parent's constructor by adding @specialization and a parameter for it.
- Override can_use_services? so it always returns true.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,187 @@
-# OOP-school-library
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="https://www.alamy.com/stock-photo-black-and-white-library-logotype-on-white-background-school-vector-149396619.html" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (rotimiazeez) [inline README.md]: Please ensure the logo provided is not broken 👍🏽
Screenshot from 2023-04-17 15-20-15

File: README.md [hunk 2]
```diff
@@ -1 +1,187 @@
...
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
+
```
Comment (rotimiazeez) [inline README.md]: Please note that the MIT link provided is broken. Kindly fix it 👍🏽 
Screenshot from 2023-04-17 15-17-53

File: person.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  end
+
+  attr_accessor :id, :name, :age, :parent_permission
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+end
```
Comment (rotimiazeez) [inline person.rb]: Please ensure a public method `can_use_services?` that returns true if a person is of age or has permission from parents is added to your person.rb file.
  
```
    def can_use_services? 
    of_age?() || @parent_permission
  end
```

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @chepkok3 👍🏽,
## Good job so far!
### To Highlight 👇🏽
- No Linter error ✔️ 
- Good PR description 👍🏽 
- Good commit message(s) 😉
- Descriptive README 📖
You still need to work on some issues to prepare your project for the final evaluation, but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR WORDS>


## Suggested changes
_Check the comments under the review._

Please feel free to use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend considering them as they can improve your code._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something needs to be 100% clear.
Remember to tag (@rotimiazeez) me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

File: person.rb (hunk 3 above)
Comment (chepkok3) [reply person.rb]: @rotimiazeez Thanks for highlighting them. I will fix

**Review by afanimartin — APPROVED** [review]
Hi @chepkok3, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- MIT license link fixed
- Person.rb updated as requested

# Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

## Find me on slack
### Martin Afani

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**